### DISCUSSION TITLE
[Projects Beta] Add images to draft issues

### DISCUSSION THREAD

**Discussion by davidjb99** [community/community#9917] [Projects and Issues]
It does not appear to be possible to add images to draft issues, this would be very useful for uploading screenshots etc...

**Comment by drone1** [comment]
Also want this. Would like to be able to drag an image and attach it, for starters. Doesn't even need to be inline.

Or click a button to attach 1 image.

**Comment by rmcsharry** [comment]
> It does not appear to be possible to add images to draft issues, this would be very useful for uploading screenshots etc...

![image](https://user-images.githubusercontent.com/1156100/162616162-4d15efb0-3c28-499c-aea9-5f46e76381ac.png)

**Comment by Sammeeey** [comment]
### This feedback has at least two duplicates #14920 and #13188
@davidjb99 I would really appreciate if you could improve your initial description by editing it and including the description from #13188.
You are also welcome to insert my description below as well.
Here is the raw code, which you can just copy+paste:
```
### Visualization of issue
- manual insertion of images (with URLs) possible
- copy+paste insertion (without URLs) impossible
![grafik](https://user-images.githubusercontent.com/49591562/164958924-d23696a4-f241-4b56-9525-dd5e2a47f007.png)
![grafik](https://user-images.githubusercontent.com/49591562/164959033-58f1b71b-dd9a-4797-b7b4-9f0a8e63f2bc.png)


### Why is this an issue?
Sometimes you just have images/screenshots in the clipboard (which don't have URL's).
Hence we need a solution to insert these images too✌
```

### Visualization of issue
- manual insertion of images (with URLs) possible
- copy+paste insertion (without URLs) impossible
![grafik](https://user-images.githubusercontent.com/49591562/164958924-d23696a4-f241-4b56-9525-dd5e2a47f007.png)
![grafik](https://user-images.githubusercontent.com/49591562/164959033-58f1b71b-dd9a-4797-b7b4-9f0a8e63f2bc.png)


### Why is this an issue?
Sometimes you just have images/screenshots in the clipboard (which don't have URL's).
Hence we need a solution to insert these images too✌
